### DISCUSSION TITLE
PRs are not showing up on Read Me Stats (Fixed)

### DISCUSSION THREAD

**Discussion by AnaheimDucks16** [anuraghazra/github-readme-stats#1071] [Q&A] [answered]
My read me stats show I have 0 pull requests but on Github, it shows I have multiple pull requests.
Read Me Stats: 
![image](https://user-images.githubusercontent.com/75493777/119070738-6d519800-b99d-11eb-9de6-2da509960913.png)
Github: 
![image](https://user-images.githubusercontent.com/75493777/119070784-822e2b80-b99d-11eb-83e0-4e8539a95361.png)

**Comment by AnaheimDucks16** [comment, marked as answer]
Fixed it

**Reply by analizrocha** [reply]
how do you fixed it?

**Reply by AnaheimDucks16** [reply]
Its been a while so I kind of forgot

**Reply by rickstaa** [reply]
@analizrocha If you have any problems with your stats card feel free to open a new discussion.
